webui/tests/tests/StepImplementation.ts

```diff
@@ -45,9 +46,14 @@ const HOST = process.env.DET_MASTER || 'localhost:8080';
 const BASE_PATH = process.env.PUBLIC_URL || '/det';
 const BASE_URL = `${HOST}${BASE_PATH}`;
 const SCREENCAST = process.env.SCREENCAST === 'true';
+const viewports = {
+  mobile: { width: 411, height: 823 },
+  desktop: { width: 960, height: 1536 },
```

Comment:
**suggestion:** this seems a bit on the smaller side for the width. One study done on the [most common browser window size](https://css-tricks.com/screen-resolution-notequalto-browser-window/). TL;DR ~ 1366×768

Reply:
I just used what you set for cypress tests in your PR. we should probably use the same on both? 1366×768 sg